pkg/common/events.go

```diff
@@ -56,62 +58,109 @@ var ValidEvents = []gh.Event{
 	gh.ProjectEvent,
 	gh.PublicEvent,
 	gh.PullRequestEvent,
-	gh.PullRequestReviewEvent,
 	gh.PullRequestReviewCommentEvent,
+	gh.PullRequestReviewEvent,
 	gh.PushEvent,
 	gh.ReleaseEvent,
 	gh.RepositoryEvent,
+	gh.RepositoryVulnerabilityAlertEvent,
+	gh.SecurityAdvisoryEvent,
 	gh.StatusEvent,
-	gh.TeamEvent,
 	gh.TeamAddEvent,
+	gh.TeamEvent,
 	gh.WatchEvent,
 }
 
-// SubjectFromGitHubEvent computes the CE subject from GitHub event
-func SubjectFromGitHubEvent(gitHubEvent gh.Event, payload interface{}, logger *zap.SugaredLogger) string {
+// SubjectAndExtensionsFromGitHubEvent computes the CE subject from GitHub event
+func SubjectAndExtensionsFromGitHubEvent(gitHubEvent gh.Event, payload interface{}, logger *zap.SugaredLogger) (string, map[string]interface{}) {
 	// The decision of what to put in subject is somewhat arbitrary here (i.e., it's the author's opinion)
 	// TODO check if we should be setting subject to these values.
 	var subject string
+	var exts = make(map[string]interface{})
 	var ok bool
 	switch gitHubEvent {
+	case gh.CheckRunEvent:
+		var c gh.CheckRunPayload
+		if c, ok = payload.(gh.CheckRunPayload); ok {
+			subject = strconv.FormatInt(c.CheckRun.ID, 10)
+			exts["owner"] = c.Repository.Owner.Login
+			exts["repository"] = c.Repository.Name
+			//exts["installation"] = c.Installation.ID
```

Comment:
the reason they are there and commented out is the GitHub API supports them, but the client being used does not (yet). Since it's not everywhere, I added and commented out. I'm okay with removing these, but they were put in specific places for a reason (it's not present in some calls).

Reply:
Sure but commented code is like dead code. It serves no purpose and only you knows why they are here.